.github/workflows/docker-devito.yml

```diff
@@ -22,13 +22,13 @@ jobs:
             tag: 'nvidia-nvc'
             flag: '--gpus all'
             test: 'tests/test_gpu_openacc.py tests/test_gpu_common.py'
-            runner: ["self-hosted", "gpu", "docker", "v1004"]
+            runner: ["self-hosted", "nvidiagpu"]
 
           - base: 'bases:nvidia-clang'
             tag: 'nvidia-clang'
             flag: '--gpus all'
             test: 'tests/test_gpu_openmp.py tests/test_gpu_common.py'
-            runner: ["self-hosted", "gpu", "docker", "kimogila"]
+            runner: ["self-hosted", "nvidiagpu", "docker"]
```

Comment:
`amdgpu` below?

Reply:
yes

and why do we need "docker" here and not in the one above?